llvm/lib/Target/AMDGPU/VOP1Instructions.td

```diff
@@ -384,7 +403,12 @@ class VOP_MOVREL<RegisterOperand Src1RC> : VOPProfile<[untyped, i32, untyped, un
 
   let Outs = (outs);
   let Ins32 = (ins Src0RC32:$vdst, Src1RC:$src0);
-  let Ins64 = (ins Src0RC64:$vdst, Src1RC:$src0);
+
+  let Ins64 = !con((ins Src0RC64:$vdst),
```

Comment:
Instead of using !if, can you override the relevant fields such as  Ins64 and Src0Mod in VOP_MOVRELD?